deploy-extension.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Isn't this effectively doing the same thing as `/test/manual_test.py`? Is there a reason to prefer this over that?

Reply:
No, there isn't. I just forgot `test/manual_test.py` already does this

Reply:
I have deleted it